### DISCUSSION TITLE
Variable particle resolution

### DISCUSSION THREAD

**Discussion by GUT2060** [InteractiveComputerGraphics/SPlisHSPlasH#313] [Q&A]
Hi ! 
I'm wondering if you are planning on adding a variable particle resolution feature in the up-coming future. if not ,which parts do you think will need modifications and what would be the best approach doing this ? 
Thanks

**Comment by janbender** [comment]
Yes, but it will take a while until we have this feature. The first step was the development of a neighborhood search which supports this feature.
